doc/2/guides/getting-started/2-storing-and-accessing-data/index.md

```diff
@@ -0,0 +1,207 @@
+---
+code: false
+type: page
+title: Storing and accessing Data
+description: Create and retrieve documents
+order: 200
+---
+
+# Storing and accessing Data
+
+Now that Kuzzle is running, we are going to create some documents.
+
+Kuzzle organizes the data storage in 4 levels: 
+  - indexes
+  - collections
+  - documents
+  - fields
+
+An index brings together several collections, which in turn contain several documents, each of which is composed of several fields.
+
+The collections have mappings that characterize the way Elasticsearch will index the data for the search.
```

Comment:
maybe add a link to know more about the mappings, either from our doc, or directly to the elasticsearch doc